clever/www/index.html

```diff
@@ -1,4 +1,4 @@
-<h1>CLEVER Drone Kit Tools</h1>
+<h1>CLEVER Drone Kit Tools <!-- $VERSION --></h1>
```

Comment:
That would read as "Drone Kit Tools v0.19", as if this is only the tools version, not the image version itself. That is not technically incorrect, but would be somewhat misleading.